package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "jest --coverage",
     "build": "webpack",
-    "lint": "npx eslint ./src"
+    "lint": "npx eslint ./src",
+    "lint--fix": "npx eslint --fix ./src"
```

Comment:
sounds good hahaha
but you could create with the key equal `lint:fix`

Reply:
i did that